src/test/java/org/minions/devfund/mauricioz/linkedList/InsertSortedTest.java

```diff
@@ -0,0 +1,144 @@
+    public void setUp() throws Exception {
+    }
+
+    /**
+     * test.
+     */
+    @Test
+    public void testInsertMiddle() {
+        LinkedList var1 = new LinkedList();
+        var1.add(3);
+        var1.addLast(8);
+        var1.addLast(11);
+
+        try {
+            LinkedListUtils.insertSorted(var1, 10);
+        } catch (Exception var3) {
+            Assert.fail("insertSorted throws " + var3 + " when placing value in middle of list");
+        }
+
+        Assert.assertTrue("insertSorted creates LinkedList of incorrect size when placing value in middle of list", var1.size() == 4);
+        Assert.assertTrue("insertSorted does not correctly insert element when placing value in middle of list", (Integer)var1.get(0) == 3);
+        Assert.assertTrue("insertSorted does not correctly insert element when placing value in middle of list", (Integer)var1.get(1) == 8);
+        Assert.assertTrue("insertSorted does not correctly insert element when placing value in middle of list", (Integer)var1.get(2) == 10);
+        Assert.assertTrue("insertSorted does not correctly insert element when placing value in middle of list", (Integer)var1.get(3) == 11);
+    }
+
+    /**
+     * test.
+     */
+    @Test
+    public void testInsertEnd() {
+        LinkedList var1 = new LinkedList();
+        var1.add(3);
+        var1.addLast(8);
+        var1.addLast(11);
+
+        try {
+            LinkedListUtils.insertSorted(var1, 14);
+        } catch (Exception var3) {
+            Assert.fail("insertSorted throws " + var3 + " when placing value at end of list");
+        }
+
+        Assert.assertTrue("insertSorted creates LinkedList of incorrect size when placing value at end of list", var1.size() == 4);
+        Assert.assertTrue("insertSorted does not correctly insert element when placing value at end of list", (Integer)var1.get(0) == 3);
+        Assert.assertTrue("insertSorted does not correctly insert element when placing value at end of list", (Integer)var1.get(1) == 8);
+        Assert.assertTrue("insertSorted does not correctly insert element when placing value at end of list", (Integer)var1.get(2) == 11);
+        Assert.assertTrue("insertSorted does not correctly insert element when placing value at end of list", (Integer)var1.get(3) == 14);
+    }
+
+    /**
+     * test.
+     */
+    @Test
+    public void testInsertFront() {
+        LinkedList var1 = new LinkedList();
+        var1.add(3);
+        var1.addLast(8);
+        var1.addLast(11);
+
+        try {
+            LinkedListUtils.insertSorted(var1, 1);
+        } catch (Exception var3) {
+            Assert.fail("insertSorted throws " + var3 + " when placing value at front of list");
+        }
+
+        Assert.assertTrue("insertSorted creates LinkedList of incorrect size when placing value at front of list", var1.size() == 4);
+        Assert.assertTrue("insertSorted does not correctly insert element when placing value at front of list", (Integer)var1.get(0) == 1);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define a constant instead of duplicating this literal "insertSorted does not correctly insert element when placing value at front of list" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)